### PR TITLE
[WIP] Hotfix add language at filter rule

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix add language at filter rule...

...but a better way would be to adjust the filter settings with `all_langs` https://github.com/MetaModels/filter_text/issues/29

### CODE REVIEW

**Comment by zonky2** [comment]
@discordier pls review the PR

File: contao/config/event_listeners.php [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?php
+
 /**
- * This file is part of MetaModels/filter_text.
+ * This file is part of MetaModels/attribute_translatedurl.
```
Comment (discordier) [inline contao/config/event_listeners.php]: This is still `MetaModels/filter_text` :smile_cat:

File: src/MetaModels/Filter/Setting/Text.php [hunk 2]
```diff
@@ -264,4 +267,18 @@ private function addFilterParam($strParam)
...
+     *
+     * @param IMetaModel $objMetaModel The metamodel.
+     *
+     * @return array|null|\string[]
+     */
+    private function getAvailableLanguages(IMetaModel $objMetaModel)
+    {
+        return ($objMetaModel->isTranslated() && $this->get('all_langs'))
+            ? $objMetaModel->getAvailableLanguages()
+            : array($objMetaModel->getActiveLanguage());
```
Comment (discordier) [inline src/MetaModels/Filter/Setting/Text.php]: We can now use short array here

**Comment by zonky2** [comment]
done

**Pull request merged**